app/renderer/css/servermanager.css

```diff
@@ -0,0 +1,86 @@
+html, body {
+    height: 100%;
+    margin: 0;
+}
+
+#content {
+    display: flex;
+    height: 100%;
+}
+
+#sidebar {
+    background: #222c31;
+    width: 88px;
+    padding: 40px 0 20px 0;
+    justify-content: space-between;
+    display: flex;
+    flex-direction: column;
+    -webkit-app-region: drag;
+}
+
+#webview {
+    width: 100%;
+}
+
+.action-button {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 10px;
+}
+
+.action-button i {
+    color: #6c8592;
+    font-size: 28px;
+}
+
+
+#servers-container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.server-button {
+    position: relative;
+    margin: 5px 0 5px 6px;
+}
+
+.server-button.active::before{
+    content: "";
+    background: #43bba6;
+    border-radius: 3px;
+    width: 6px;
+    position: absolute;
+    height: 6px;
+    left: -12px;
+    top: 25px;
+}
+
+.server-button .server-name{
+    background: #a4d3c4;
+    border-radius: 22px;
+    width: 44px;
+    height: 44px;
+    position: relative;
+    margin: 5px 0;
+    z-index: 11;
+    line-height: 44px;
+    font-size: 24px;
+    font-family: sans-serif;
+    color: #194a2b;
+    text-align: center;
+    overflow: hidden;
+
+}
+
+.server-button.active .server-ring {
+    background: #43bba6;
+    border-radius: 24px;
+    width: 48px;
+    height: 48px;
+    position: absolute;
+    left: -2px;
+    top: 3px;
+    z-index: 10;
+}
```

Comment:
A newline is missing here.  We should consider making our linter check for this.